plugins/woocommerce/tests/e2e-pw/bin/test-helper-apis.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * Plugin Name: Test Helper APIs
+ * Description: Utility REST API designed for E2E testing purposes. Allows turning features on or off, and setting option values
+ */
+function register_helper_api() {
+	register_rest_route(
```

Comment:
APIs for setting feature flags or setting wordpress options directly. This plugin is installed and activated in setup of wp env environment